client/extensions/woocommerce/state/sites/settings/actions.js

```diff
@@ -80,53 +79,69 @@ export const doInitialSetup = (
 		},
 	];
 
+	// TODO Only enable taxes when applicable
+	update = update.concat( [
+		{
+			group_id: 'general',
+			id: 'woocommerce_calc_taxes',
+			value: 'yes',
+		},
+	] );
+
 	if ( pushDefaultsForCountry ) {
+		// TODO Support other currency positions, post-v1 etc. See https://github.com/Automattic/wp-calypso/issues/15498
```

Comment:
It might be worth a comment like

> in the event this is a brand new site, WooCommerce defaults for that country will be automatically used in the wizard.

so it makes it clear that other countries will still get their correct values though the wizard.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/21297/commits/20e50be7b870bb2e5ff1c9223c1c55de64434163